### PR TITLE
Add design methodology

### PR DESCRIPTION
**Pull request by MSakamaki**
#154

### CODE REVIEW

File: src/templates/en/2019/methodology.html [hunk 1]
```diff
@@ -3,12 +3,109 @@
...
+        <li>Lorem Ipsum Content</li>
+      </ul>
+    </div>
+  </nav>
 
-<h2>Overview of HTTP Archive</h2>
+  <section class="content">
+    <section class="intro">
+      <h1><div class="decorative-line"></div>Methodology<div class="decorative-line-mobile"></div></h1>
+      <p>A concise map into the guidelines that are strictly followed to output our reports.</p>
```
Comment (rviscomi) [inline src/templates/en/2019/methodology.html]: I think this section is more of a subtitle than paragraph. The font appears similar to the title in the design:

image

For comparison:

image

I think the difference is that titles have the Poppins font while paragraphs use Lato.

File: src/static/css/methodology.css [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+}
+.decorative-line-mobile {
+ display: none; 
+}
+.floating-card {
+  border-radius: 16px;
+  box-shadow: 0 0 5px 5px #f6f7f9;
+}
+.code-block {
+  margin: 8px;
```
[the remark this answers is not in the document]
Comment (MSakamaki) [reply src/static/css/methodology.css]: Certainly, margin is unnecessary, so set it wider.

File: src/templates/en/2019/methodology.html [hunk 3]
```diff
@@ -3,12 +3,109 @@
...
+GROUP BY
+    yyyy
+ORDER BY
+    zzzz
+          </code>
+      </div>
+    </section>
+    <section> 
+      <h3>Lorem Ipsum Title H3</h3>
+      <img class="methodology-banner" src="/static/images/methodology-layer.png" alt="Methodology">
```
[the remark this answers is not in the document]
Comment (MSakamaki) [reply src/templates/en/2019/methodology.html]: ok, delete img tag and `methodology-layer.png`.

File: src/static/css/methodology.css [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+.code-block code {
+  display: block;
+  white-space: pre-wrap  
+}
+
+.index {
+  grid-area: index;
+}
+.index .index-box {
+  margin: 8px;
```
[the remark this answers is not in the document]
Comment (MSakamaki) [reply src/static/css/methodology.css]: >On mobile this should have the same margins as the header

Thank you, was a new practices for me.
I will try to align the margin..

スクリーンショット 2019-10-17 16 02 30

File: src/templates/en/2019/methodology.html [hunk 5]
```diff
@@ -3,12 +3,109 @@
...
+ 
+{% block styles %}
+{{ super() }}
+<link rel="stylesheet" href="/static/css/methodology.css">
+{% endblock %}
 
 {% block main %}
-<h1>Methodology</h1>
-<p>TODO</p>
+<article id="methodology">
```
[the remark this answers is not in the document]
Comment (MSakamaki) [reply src/templates/en/2019/methodology.html]: @rviscomi 
OK, I'll fix it in methodology branch.

File: src/templates/en/2019/methodology.html (hunk 1 above)
Comment (MSakamaki) [reply src/templates/en/2019/methodology.html]: Oh, I overlooked it.
I will fix it!

File: src/templates/en/2019/methodology.html [hunk 6]
```diff
@@ -3,12 +3,109 @@
...
+  <nav class="index">
+    <div class="index-box floating-card">
+      <h2 class="header">Index</h2>
+      <h2 class="header-mobile">Index</h2>
+      <ul>
+        <li class="active">Lorem Ipsum Content
```
[the remark this answers is not in the document]
Comment (MSakamaki) [reply src/templates/en/2019/methodology.html]: complited !

File: src/templates/en/2019/methodology.html [hunk 7]
```diff
@@ -3,12 +3,109 @@
...
+
+{% block scripts %}
+{{ super() }}
+<script nonce="{{ csp_nonce() }}">
+
+var indexBox = document.querySelector('.index-box');
+var indexBoxTitle = document.querySelector('.index-box h2.header-mobile');
+
+indexBoxTitle.addEventListener('click', function(e) {
+  if (indexBox.classList.contains('show')) {
```
[the remark this answers is not in the document]
Comment (MSakamaki) [reply src/templates/en/2019/methodology.html]: complited !

File: src/templates/en/2019/methodology.html [hunk 8]
```diff
@@ -3,12 +3,109 @@
...
+    </section>
+    <section> 
+      <div class="code-block floating-card">
+          <h3>Lorem Ipsum Title H3</h3>
+          <div class="divider"></div>
+<!-- This is just an example of embedding a code sample, you can disregard for now.
+https://github.com/HTTPArchive/almanac.httparchive.org/issues/130#issuecomment-526694637 -->
+          <code>
```
[the remark this answers is not in the document]
Comment (rviscomi) [reply src/templates/en/2019/methodology.html]: Great question. Let's not do syntax highlighting. If anything I can manually style any demo queries.

**Pull request merged**